### ISSUE TITLE
Add perception value to the tracker

### ISSUE THREAD

**Issue by Mustungun** [314r-app/pf2eBattleTracker_issues#12]
The title says it all. Perception is used almost as much as saving throws. It's very useful to have for PCs and enemies alike. The good part is how the system already tracks perception, so it should be easy to add. The bad part is understanding how this feature could make the UI crowded, especially when using a phone in portrait mode. I don't see the perfect compromise here... maybe making it toggleable? I don't know... but it would be very useful to have it. Let's talk about it!

**Comment by 314r-app** [comment]
Hello and thank you for your interest in my application. 

It is indeed a good suggestion, the only difficulty to implement will be in the design of the UI. First an easy update will be to change the menu that open when clicking on the name of a character and add a box for the perception.

To display in on the tracker screen directly I think that it is not possible to add in on the same line as the defensive stat (AC and saves), because on a 5" portrait screen this line already take all the space available. And reducing the font will make it too hard to read. 
Make another separate line seems too much, except maybe if I want to add more information in the future. The only space that I see would be on the first line, between the name and the HP (flushed to the right, so it's always aligned just before the HP). With a little "eye" icon before. What do you think about this placement ? 

In the last updated I finally introduced a "setting" menu and I planned to use to allow customization of the UI. Making the perception toogleable is definitely possible and I'll add it.

**Comment by Mustungun** [comment]
I agree with you. That seems to be the best option. You're going to have numbers close to each other, which could make the tidiness a bit awkward. If you struggle too much to make it work in a satisfying way, you could also consider to print the value to the left edge of the screen, even left of the creature name. In my mind, this option looks worse, but I'm trusting your judgement. It may be a good idea to try it out anyway and see which one looks better

**Comment by 314r-app** [comment]
I am adding that to my to-do list. When I will be done with the current feature I'll make some tests and post screenshots here.

**Comment by Mustungun** [comment]
Alright. Thank you! Looking forward to it!
